scripts/dummy_worker.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+// eslint-disable-next-line node/no-unsupported-features/node-builtins
```

Comment:
not required after https://github.com/hexojs/hexo-util/pull/212/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R57
comment in other files can be removed too.